cluster-operator/src/main/java/io/strimzi/operator/cluster/model/KafkaCluster.java

```diff
@@ -476,6 +478,13 @@ public static KafkaCluster fromCrd(Kafka kafkaAssembly, KafkaVersion.Lookup vers
                 log.warn("Your desired Kafka storage configuration contains changes which are not allowed. As a " +
                         "result, all storage changes will be ignored. Use DEBUG level logging for more information " +
                         "about the detected changes.");
+
+                Condition warning = StatusUtils.buildWarningCondition("KafkaStorage",
+                        "The desired Kafka storage configuration contains changes which are not allowed. As a " +
```

Comment:
I wonder if adding these status conditions means we can change the log.warns to log.debugs?

Reply:
I think it should still remain as a warning -> that might raise attention through other ways than the condition, for exxample in some log processing pipelines etc.

Reply:
Makes sense to me!